src/AMP.php

```diff
@@ -174,6 +174,17 @@ public function loadHtml($html, $options = [])
             $options['use_html5_parser'] = true;
         }
 
+        // By default the convertion of img into amp-anim is disabled (because of ressource cost)
```

Comment:
If the amp-anim is enabled, all gif and png images are downloaded by the server and processed bytewise to detect if they are animated or not.
If they are animated, the amp-anim tag is added.
If they are not animated, a normal amp-img tag is added.

This can consume a lot of resources if you don't cache the AMP output in your system.